axon-server-connector/src/main/java/org/axonframework/axonserver/connector/AxonServerConfiguration.java

```diff
@@ -663,6 +689,94 @@ public void setTimeout(long timeout) {
+
+        /**
+         * The configuration of each of the processors. The key is the name of the processor, the value represents the
+         * settings to use for the processor with that name.
+         */
+        private final Map<String, ProcessorSettings> processors = new HashMap<>();
+
+        /**
+         * Returns the settings for each of the configured processors, by name.
+         *
+         * @return the settings for each of the configured processors, by name.
+         */
+        public Map<String, ProcessorSettings> getProcessors() {
+            return processors;
+        }
+
+        public static class ProcessorSettings {
+
+            /**
+             * Configures the desired load balancing strategy for this event processor.
+             * <p>
+             * The load balancing strategy tells Axon Server how to share the event handling load among all available
+             * application instances running this event processor, by moving segments from one instance to another. Note
+             * that load balancing is <b>only</b> supported for
+             * {@link org.axonframework.eventhandling.StreamingEventProcessor StreamingEventProcessors}, as only
+             * {@code StreamingEventProcessors} are capable of splitting the event handling load in segments.
+             * <p>
+             * Defaults to {@link LoadBalancingStrategy#DISABLED}.
+             */
+            private LoadBalancingStrategy loadBalancingStrategy = LoadBalancingStrategy.DISABLED;
+
+            /**
+             * Returns the {@link LoadBalancingStrategy load balancing strategy} for this event processor. Defaults to
+             * {@link LoadBalancingStrategy#DISABLED}.
+             *
+             * @return The {@link LoadBalancingStrategy load balancing strategy} for this event processor.
+             */
+            public LoadBalancingStrategy getLoadBalancingStrategy() {
+                return loadBalancingStrategy;
+            }
+
+            /**
+             * Sets the {@link LoadBalancingStrategy load balancing strategy} for this event processor.
+             *
+             * @param loadBalancingStrategy The {@link LoadBalancingStrategy load balancing strategy} for this event
+             *                              processor.
+             */
+            public void setLoadBalancingStrategy(LoadBalancingStrategy loadBalancingStrategy) {
+                this.loadBalancingStrategy = loadBalancingStrategy;
+            }
+        }
+
+        /**
+         * The possible load balancing strategies for
+         * {@link org.axonframework.eventhandling.StreamingEventProcessor StreamingEventProcessors}.
+         */
+        public enum LoadBalancingStrategy {
+            /**
+             * Indication that load balancing is <em>disabled</em> for a given
+             * {@link org.axonframework.eventhandling.StreamingEventProcessor}.
+             */
+            DISABLED("disabled"),
```

Comment:
The options in Axon Server are dynamic. A list can be fetched of the possibilities.
As we want  any version of AF and AS to be interoperable, perhaps it's better to make this a regular string field instead of an enum, with Axon Server documentation pointing to the correct values. What do you think?

Reply:
Yea, I'd love that! However, the Axon Server documentation doesn't name the Load Balancing strategies anywhere... Dropped a question with them to figure out how to proceed.

Reply:
Regardless of this missing in the Reference Guide, I went ahead and removed the enumeration. Let's see how this pans out.